docs/data-sources/er_attachments.md

```diff
@@ -0,0 +1,77 @@
+---
+subcategory: "Enterprise Router (ER)"
+---
+
+# huaweicloud_er_attachments
+
+Use this data source to filter ER attachments within Huaweicloud.
+
+## Example Usage
+
+```hcl
+data "huaweicloud_er_attachments" "test" {
+  tags = {
```

Comment:
Missing required parameter in this usage

Reply:
fixed.